pkg/authorize/kubecertmanager/authorize.go

```diff
@@ -183,13 +310,23 @@ func (c *authorizer) isHostedInternally(ingress *extensions.Ingress) field.Error
 // isIngressPointed is responisble for checking the dns hostname is pointed to the external ingress
```

Comment:
Could probably change the comment now as its checking both

`isIngressPointed is responisble for checking the dns hostname is pointed to the expected hostname` or something

Reply:
yeah, good spot; changed